pkg/subctl/cmd/unexport.go

```diff
@@ -0,0 +1,85 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/cmd/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	mcsclient "sigs.k8s.io/mcs-api/pkg/client/clientset/versioned/typed/apis/v1alpha1"
+)
+
+var (
+	unexportCmd = &cobra.Command{
+		Use:   "unexport",
+		Short: "Stop a resource from being exported to other clusters",
+		Long:  "This command stops exporting a resource so it isn't accessible to other clusters any more",
```

Comment:
along with what @tpantelis suggested,
```suggestion
		Long:  "This command stops exporting a resource so that it isn't accessible to other clusters any more",
```
as per https://learningenglish.voanews.com/a/what-s-the-difference-so-and-so-that-/4627372.html